.github/ISSUE_TEMPLATE/release-template.md

```diff
@@ -28,15 +29,13 @@ This release process will produce releases:
       Such updates should only be included in the week before a release if there is a compelling security or stability reason to do so.
 - [ ] Wait 24 hours from last change to the issues/PRs included in FROZEN GitHub Project(s).
 - [ ] Verify target [project(s)](https://github.com/jetty/jetty.project/projects) are complete.
-- [ ] Verify that branch `jetty-10.0.x` is merged to branch `jetty-11.0.x`.
+- [ ] Verify that branch `jetty-12.0.x` is merged to branch `jetty-12.1.x`.
 - [ ] Assign issue to "build manager", who will stage the releases.
   - [ ] Create and use branches `release/<ver>` to perform version specific release work from.
   - [ ] Ensure `git fetch --tags` (as we potentially rewrite tag when re staging local tag can be out of sync and this command will fail and so fail the release script)
   - [ ] Ensure `VERSION.txt` additions for each release will be meaningful, descriptive, correct text.
-  - [ ] Stage 9.4 release with Java 11.
-  - [ ] Stage 10 release with Java 21.
-  - [ ] Stage 11 release with Java 21.
-  - [ ] Stage 12 release with Java 22. 
+  - [ ] Deploy 12 release with Java 22 to central.sonatype.com
+  - [ ] Deploy 12 release to Nexus on CI system to help with testing of release (shows up on release-staging repo)
```

Comment:
should we have some documentation about this? in a private document?

Reply:
we can automate this as part of the release script but it needs pre requisite such ssh tunnel opened or firewall to our build network.
Maybe we can make it last step of the script so it's easy to run it again manually.